config.json

```diff
@@ -0,0 +1,18 @@
+{
+    "FoodOnPlatform": 100,
+    "NumOfAgents": [2,2,2,2,2,2,2],
```

Comment:
maybe making this a little more interpretable, like `"randomAgent": int` and then in the `configparameters.go`, `func LoadParamFromJson` put it into the list in the right places? Makes it a little easier for people to input their agents etc/more readable.

Reply:
I've implemented this now, thanks.